kuiper-game/Data/Sol.solarsystem

```diff
@@ -0,0 +1,100 @@
+{
+    "position": 1,
+    "name": "Sol",
+    "distance": 0,
+    "velocity": 0,
+    "originDegrees": 0,
+    "type": "Star",
+    "satellites": [
+        {
+            "position": 1,
+            "name": "Mercury",
+            "distance": 0.387,
+            "velocity": 47.4,
+            "originDegrees": 30,
+            "type": "Planet",
+            "satellites": []
+        },
+        {
+            "position": 2,
+            "name": "Venus",
+            "distance": 0.723,
+            "velocity": 35.0,
+            "originDegrees": 170,
+            "type": "Planet",
+            "satellites": []
+        },
+        {
+            "position": 3,
+            "name": "Earth",
+            "distance": 1,
```

Comment:
It's funny how even in a Heliocentric world view, an AU is still defined as the distance between earth and our star :D

Reply:
There's a conversion if you want KM's instead :)